README.md

```diff
@@ -64,7 +64,7 @@ and renders declaratively using `lit-html`.
     * expression: ``` html`<div>${disabled ? 'Off' : 'On'}</div>` ```
     * property: ``` html`<x-foo .bar="${bar}"></x-foo>` ```
     * attribute: ``` html`<div class="${color} special"></div>` ```
-    * event handler: ``` html`<button @click="${(e) => this._clickHandler(e)}"></button>` ```
+    * event handler: ``` html`<button @click="${this._clickHandler}"></button>` ```
```

Comment:
Worth mentioning (here or elsewhere) that listener is bound to element (since that's not standard `lit-html`)?

Also, worth showing a non-decorator pattern for options?

Reply:
Please show/add as many explanations and examples as possible on things like this! :)

Reply:
The README is about to be replaced with real docs.... It already didn't say how to set up event handlers correctly, so I think this at least isn't any worse.